packages/123done/static/js/123done.js

```diff
@@ -14,16 +14,18 @@ $(document).ready(function() {
   let paymentURL;
   switch (window.location.host) {
     case '123done-latest.dev.lcip.org':
-      paymentURL = 'https://latest.dev.lcip.org/subscriptions/products/prod_Ex9Z1q5yVydhyk';
+      paymentURL =
+        'https://latest.dev.lcip.org/subscriptions/products/prod_Ex9Z1q5yVydhyk';
       break;
     case '123done-stage.dev.lcip.org':
-      paymentURL = 'TBD';
+      paymentURL =
+        'https://accounts.stage.mozaws.net/subscriptions/products/prod_FUUNYnlDso7FeB';
```

Comment:
This URL looks like the right pattern, though the product ID leads to a "no such product" error. Guessing we still need to switch to stage subhub & stripe where this product ID is available